src/Livewire/DataTables/TicketList.php

```diff
@@ -65,4 +145,84 @@ public function getFilterableColumns(string $name = null): array
     {
         return $this->availableCols;
     }
+
+    public function show(): void
+    {
+        $this->ticket = [
+            'title' => null,
+            'description' => null,
+        ];
+
+        $this->ticketTypeId = null;
+        $this->selectedAdditionalColumns = [];
+        $this->filesArray = [];
+        $this->attachments = [];
+
+        $this->showTicketModal = true;
+        $this->skipRender();
+    }
+
+    public function save(): void
+    {
+        $this->ticket = array_merge($this->ticket, [
+            'authenticatable_type' => Auth::user()->getMorphClass(),
+            'authenticatable_id' => Auth::id(),
+            'ticket_type_id' => $this->ticketTypeId,
+        ]);
+
+        if ($this->modelType && $this->modelId) {
+            $this->ticket['model_type'] = $this->modelType;
+            $this->ticket['model_id'] = $this->modelId;
+        }
+
+        try {
+            $ticket = CreateTicket::make($this->ticket)
+                ->validate()
+                ->execute();
+        } catch (\Exception $e) {
+            exception_to_notifications($e, $this);
+
+            return;
+        }
+
+        try {
+            $this->saveFileUploadsToMediaLibrary('attachments', $ticket->id, Ticket::class);
+        } catch (\Exception $e) {
+            exception_to_notifications($e, $this);
+        }
+
+        $this->notification()->success(__('Ticket created…'));
+
+        $this->showTicketModal = false;
+        $this->skipRender();
+        $this->loadData();
+    }
+
+    public function updatedAttachments(): void
+    {
+        $this->prepareForMediaLibrary('attachments');
+
+        $this->skipRender();
```

Comment:
you can remove all skipRender() as with the new data table version rendering is skipped by default after initializing the component